docs/elements.md

```diff
@@ -129,6 +131,7 @@ a `pl-answer` that has attributes:
 Attribute | Type | Default | Description
 --- | --- | --- | ---
 `correct` | boolean | false | Is this a correct answer to the question?
+`feedback` | string | — | Helper text to be displayed to the student next to the option after question is graded if this option has been selected by the student.
```

Comment:
The feedback is actually HTML, I think? That is, we template it in with triple-braces. Maybe clarify that here?
```suggestion
`feedback` | string | — | Helper text (HTML) to be displayed to the student next to the option after question is graded if this option has been selected by the student.
```

Reply:
What happens if the feedback is very long? Does it look horrible? I wonder whether we should add an option like `show-feedback-at-end` to put the feedback after the multiple choice element? Maybe I'm overthinking this, however.

Reply:
Here's what a long hint looks like:
<img width="683" alt="Screen Shot 2021-08-30 at 8 43 49 PM" src="https://user-images.githubusercontent.com/68084820/131438500-eb21d15c-9b9c-4395-9575-0b31c6cbf7ec.png">
For an example of how this looks on a smaller screen, see https://github.com/PrairieLearn/PrairieLearn/pull/4656#issuecomment-898783046

Reply:
Ok, that's kind of horrible but at least you can still read it 😄 --- I'm fine with this.